src/include/functions_common.php

```diff
@@ -510,6 +510,8 @@ function CreateReloadTimesList()
 	$tmpReloadSeconds = GetConfigSetting("ViewEnableAutoReloadSeconds", "", CFGLEVEL_USER);
 	if ( $tmpReloadSeconds > 0 )
 	{
+		//by default select predefined value
+		$_SESSION['AUTORELOAD_ID'] = 1; 
```

Comment:
I would rather recommend to add a configuration variable for this and write the default of the configuration variable into $_SESSION['AUTORELOAD_ID']. Autoreload might be desired in a later step of loganalyzer setup but not from the beginning.

https://github.com/rsyslog/loganalyzer/blob/master/src/include/config.sample.php